src/pages/Loans/LoansOverview/components/LTVMeter.tsx/LTVMeter.style.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+
+import { ReactComponent as InformationCircleIcon } from '@/assets/img/hero-icons/information-circle.svg';
+import { Span, Status, theme } from '@/component-library';
+
+type LTVLegendStatus = Exclude<Status, 'success'> | 'info';
```

Comment:
I dont think that will work. There is a LTVSection, so it would be weird having a solo LTV folder. The LTVLegend is inside the LTVMeter because it is closely related to it.